assets/index.js

```diff
@@ -4,3 +4,12 @@ import 'primereact/resources/primereact.min.css';
 if (isTouchDevice()) {
     document.documentElement.classList.add('no-touch');
 }
+
+
+window.sectionNames = {
```

Comment:
Won't this override the configurable section names from here: https://github.com/superdesk/newsroom-core/blob/develop/newsroom/templates/scripts.html#L8

Reply:
Defined in `settings.py` here https://github.com/superdesk/newsroom-core/blob/develop/newsroom/web/default_settings.py#L551

Reply:
I'll remove it, I'm only keeping it for testing until I figure out how to get it running without this. I've marked the PR as draft